### PR TITLE
Feature update docu

### PR DESCRIPTION
**Pull request by heikef**
Updated the installation section plus visualization section after receiving user feedback.

### CODE REVIEW

File: doc/installation.rst [hunk 1]
```diff
@@ -3,6 +3,18 @@
...
+Before compiling GIMIC you need to make sure that you have installed
+all of the following packages, see also the ``requirement.txt`` file.
+You need at least cython and runtest. ::
+
+* cython
+* numpy
+* pyparsing
+* pyyaml
+* sphinx
+* sphinx_rtd_theme
```
Comment (bast) [inline doc/installation.rst]: pyparsing, pyyaml, sphinx, and sphinx_rtd_theme should not be listed here. I will at some point switch to pipenv where it is clearer which packages are for running, which for testing, and which for developing.

Comment (heikef) [reply doc/installation.rst]: I included: You need at least cython and runtest -- For the time being this looks ok to me. We can always update the documentation another time.

Comment (bast) [reply doc/installation.rst]: "Inexperienced" users will however try to install these with sudo. I think we should really remove these from this list.

Comment (heikef) [reply doc/installation.rst]: Maybe we should recommend to install Anaconda2 and use conda install XXX to get all the packages. Inexperienced and pragmatic users would like this.

Comment (heikef) [reply doc/installation.rst]: Just to be clear: I do not mind removing the stuff from the list. Instead I try to figure out a user friendly installation way.

Comment (bast) [reply doc/installation.rst]: I agree.

Comment (heikef) [reply doc/installation.rst]: Would it be acceptable to recommend installation of Anaconda2 then? If yes I will update the documentation and file a new pull request.

Comment (bast) [reply doc/installation.rst]: Yes I think this is a good idea. Better than letting users sudo install things.

**Comment by bast** [comment]
One more nit-pick: it's `requirements.txt` instead of `requirement.txt`.

**Comment by heikef** [comment]
Sorry. I will fix this.

**Comment by bast** [comment]
Thumbs up! Do you want me to squash-merge?

**Comment by heikef** [comment]
No, I will do it myself.

**Comment by bast** [comment]
Super - thank you!

**Pull request merged**